build/api/ui.api.md

```diff
@@ -2015,7 +2015,7 @@ export interface MenuItemPropsTitleRequired<T> {
 // @public (undocumented)
 export type MenuProps = HTMLDivElementProps & {
     className?: NestedClassName;
-    caret?: ReactNode;
```

Comment:
isn't react node more flexible?

Reply:
Menu is just a container. Not it only toggles CSS `display:none`. It would require React context to enable passing `ReactNode` down to children.